storage/entities/base_entity.js

```diff
@@ -321,7 +310,7 @@ class BaseEntity {
 		let filtersObject = filters;
 
 		if (Array.isArray(filters)) {
-			filtersObject = Object.assign({}, ...filters);
+			filtersObject = { ...filters };
```

Comment:
This won't work. You're deconstructing an array and creating an object of objects.

Maybe the following will do the trick?
```
filters.reduce((acc, curr) =>
	({ ...acc, ...curr }),
	{}
);
```